packages/web-components/src/components/va-text-input/test/va-text-input.e2e.ts

```diff
@@ -15,7 +15,7 @@ describe('va-text-input', () => {
             Hello, world
           </label>
           <slot></slot>
-          <span id="error-message" role="alert"></span>
+          <span id="input-error-message" role="alert"></span>
```

Comment:
I updated this id to match what the [USWDS example](https://designsystem.digital.gov/components/text-input/) uses: 

![Screenshot 2023-02-07 at 10 40 33 AM](https://user-images.githubusercontent.com/872479/217307563-32a8f6fc-17ae-437f-aac8-a21d42840aae.png)